examples/fluids/src/setupts.c

```diff
@@ -21,23 +20,23 @@ PetscErrorCode ComputeLumpedMassMatrix(Ceed ceed, DM dm, CeedData ceed_data, Vec
   PetscFunctionBeginUser;
 
   // CEED Restriction
-  CeedElemRestrictionGetNumComponents(ceed_data->elem_restr_q, &num_comp_q);
-  CeedElemRestrictionGetNumComponents(ceed_data->elem_restr_qd_i, &q_data_size);
-  CeedElemRestrictionCreateVector(ceed_data->elem_restr_q, &m_ceed, NULL);
-  CeedElemRestrictionCreateVector(ceed_data->elem_restr_q, &ones_vec, NULL);
+  CeedElemRestrictionGetNumComponents(ceed_data->er_q, &num_comp_q);
+  CeedElemRestrictionGetNumComponents(ceed_data->er_qd_i, &q_data_size);
+  CeedElemRestrictionCreateVector(ceed_data->er_q, &m_ceed, NULL);
+  CeedElemRestrictionCreateVector(ceed_data->er_q, &ones_vec, NULL);
   CeedVectorSetValue(ones_vec, 1.0);
 
   // CEED QFunction
-  CeedQFunctionCreateInterior(ceed, 1, Mass, Mass_loc, &qf_mass);
+  PetscCall(CreateMassQFunction(ceed, num_comp_q, &qf_mass));
   CeedQFunctionAddInput(qf_mass, "q", num_comp_q, CEED_EVAL_INTERP);
   CeedQFunctionAddInput(qf_mass, "qdata", q_data_size, CEED_EVAL_NONE);
   CeedQFunctionAddOutput(qf_mass, "v", num_comp_q, CEED_EVAL_INTERP);
```

Comment:
I think setting the inputs and output can be folded into `CreateMassQFunction`, which would make the calls simpler.

Reply:
Done!